i18n/zh/docusaurus-plugin-content-docs/current/advanced/debug.md

```diff
@@ -1,71 +1,70 @@
 ---
-title: 启用Kubectl logs/exec/attach等能力
+title: 启用 Kubectl logs/exec/attach 边端运行的 pod
 sidebar_position: 3
 ---
 
-## Prepare certs
```

Comment:
Seems this feature now is enabled by default on cloud. We just need turn on edgecore config and restart it.

Reply:
Yes, when verifying with the latest version, the cloud is enabled by default. To make the configuration applicable for both new and old versions, the relevant prompts have been added to the configuration section.